globalize-strings/README.md

```diff
@@ -0,0 +1,16 @@
+## Gloablize strings
+The script does the following:
+1. Git clone the repos specified in `repos.json` to the current directory
+2. Go through each downloaded repo and run `slt-globalize`.  
+3. The output text is saved under `output.txt`. 
+
+## Installation
+Make sure you have the following packages installed:
+1. `git-walk` (https://www.npmjs.com/package/git-walk)
+2. `strong-globalize` (https://www.npmjs.com/package/strong-globalize)
+
+## Running the script
+Just run the script:
+```
+./globalize.sh
```

Comment:
Does it _need_ to work on Windows? This is just a script that we run ourselves (or on a Docker container, if we want to automate the process).

Reply:
I meant it would be better if we can make it run on windows as well, in case someone does not posses a non-windows machine. Anyways, I didn't disapprove the pr. It was more of a nitpick!

Reply:
The script is written in Unix Shell/Bash syntax, it won't run in `cmd.exe` even without `./` prefix.

I would personally recommend Windows developers to enable Windows Subsystem for Linux, or maybe use Git for Windows (which comes with cygwin & bash), and then run this script from that environment.

AFAIK, there is no truly native `git` client available for Windows, so any solution need to invoke `git` will require the Windows user to have basic Unix stuff installed in order to make git work, in which case I think it's ok to tell them to run this script from the Unix shell included in `git` distro.

Reply:
Thanks for the comments.  Since it's only for internal consumption, I'd add a line that it will only run in unix shell.